### ISSUE TITLE
proc/instances does not found in the filesystem which has mounted.

### ISSUE THREAD

**Issue by parkour99** [28msec/gridfs-fuse#3]
Hi,
I am trying to run your gridfs-fuse on ubuntu14.04 with the driver in the ubuntu repository ,but i don't know how to use memcached  in your implement version. 
for example, from your readme .

(1) ./bin/gridfs foobar -o mongo_db=fuse -f
(2) For the Memcached Administration,
you said : ls foobar/proc/instances
my question is :how i create or find the proc,or proc/instances in the path which has mounted gridfs-fuse filesystem.
In a nutshell, for the memcached ,i don't know how to use it on this filesystem.
Your reply will be appreciated. thanks.

**Comment by dknochen** [comment]
Do you have memcached running on 11211?
Have you successfully mounted gridfs-fuse to foobar?
If so, then what does `ls foobar/proc/instances` return?

**Comment by parkour99** [comment]
yes, I have mounted gridfs-fuse successfully. but i mounted the location is called 'mongofs' folder.
so the following data is my running result.
1, memcached is running on 11211, result as follow:
hehe@hehe02:~/develop$ netstat -tlnp|grep 11211
tcp        0      0 127.0.0.1:11211         0.0.0.0:*               LISTEN      -
               
2, i touch a 192.168.101.54:11211 server. but i can not find the log info adding server successfully  with command :tail -f /var/log/syslog . Running result as follow:

hehe@hehe02:~/develop$ ll mongofs/proc/instances/
总用量 0
d--------- 0 hehe hehe 0  1月  1  1970 ./
d--------- 0 hehe hehe 0  1月  1  1970 ../
---------- 0 hehe hehe 0  1月  1  1970 192.168.101.54:11211
so how should i know the memcached  worked in the gridfs-fuse? 
wait for your answer online.thanks for your reply.
